src/lib/server.ts

```diff
@@ -79,8 +74,26 @@ export const buildServer: (options: {
       },
     },
   });
+  let redis: Redis.Redis | undefined;
+
+  // eslint-disable-next-line prefer-const
+  redis = new Redis(redisUrl, {
+    connectionName: `stadtpuls-api-${stage}`,
+
+    autoResubscribe: false,
+    // lazyConnect: true,
+    connectTimeout: 500,
+    maxRetriesPerRequest: 0,
+    enableOfflineQueue: false,
+  });
+  redis.on("error", (err) => {
+    // TODO: [STADTPULS-726] Remove console log when done
```

Comment:
It seems you forgot a comment above and a TODO comment + console.log

Reply:
Leaving this in here was intentional. :)